elements/lisk-tree/src/merkle_tree/merkle_tree.ts

```diff
@@ -24,41 +24,57 @@ import {
 	LEAF_PREFIX,
 	BRANCH_PREFIX,
 } from './constants';
-import { NodeData, NodeInfo, NodeType, NodeSide, Proof } from './types';
+import { InMemoryDB } from './inmemory_db';
+import { PrefixStore } from './prefix_store';
+import { NodeData, NodeInfo, NodeType, NodeSide, Proof, Database } from './types';
 import { generateHash, getBinaryString, isLeaf, getPairLocation } from './utils';
 
+const hashToValuePrefix = Buffer.from([0]);
+const locationToHashPrefix = Buffer.from([1]);
```

Comment:
Can we make these as constants and move to constants file? etc `LOCATION_TO_HASH_PREFIX`